protected/humhub/libs/DynamicConfig.php

```diff
@@ -19,17 +19,6 @@
  */
 class DynamicConfig extends BaseObject
```

Comment:
@gevorgmansuryan We should probably also rename the file to e.g. “DatabaseCredConfig” in a next PR and further simplify it.

Reply:
not sure we need `load()` here. Maybe just a `saveCurrent()`.